include/picongpu/plugins/IntensityPlugin.hpp

```diff
@@ -89,24 +89,24 @@ struct KernelIntensity
 
         if (threadId.x() == 0)
         {
-            /*clear destination arrays*/
+            // clear destination arrays
             s_integrated[threadId.y()] = float_X(0.0);
             s_max[threadId.y()] = float_X(0.0);
         }
         __syncthreads();
 
-        /*move cell wise over z direction(without garding cells)*/
+        // move cell-wise over z direction(without garding cells)
         for (int z = GuardSize::z::value * SuperCellSize::z::value; z < cellsCount.z() - GuardSize::z::value * SuperCellSize::z::value; ++z)
         {
-            /*move supercell wise over x direction without guarding*/
+            // move supercell-wise over x direction without guarding
             for (int x = GuardSize::x::value * SuperCellSize::x::value + threadId.x(); x < cellsCount.x() - GuardSize::x::value * SuperCellSize::x::value; x += SuperCellSize::x::value)
```

Comment:
We changed in the past to use `supercell` in comment like in our papers. Since the beginning of the code writing we wrote `SuperCell` very where in the code.
Maybe we should change (not in this PR) everywhere in the code `SuperCell` to `Supercell` before the 0.4.0 release because we have a lot of braking changes.
@ax3l What do you think?

Reply:
I see. And I think even leaving as is is acceptable, I just did not know the backstory of it.

Reply:
I think leaving is ok here as well for now